index.bs

```diff
@@ -940,7 +940,13 @@ NOTE: The <a href="https://immersive-web.github.io/layers">WebXR layers module</
 When an {{XRSession}} |session| receives updated [=viewer=] state for timestamp |frameTime| from the [=XRSession/XR device=], it runs an <dfn>XR animation frame</dfn>, which MUST run the following steps regardless of if the [=list of animation frame callbacks=] is empty or not:
 
   1. Let |now| be the [=current high resolution time=].
-  1. Let |frame| be a [=new=] {{XRFrame}} with [=XRFrame/time=] |frameTime| and {{XRFrame/session}} |session| in the [=relevant realm=] of |session|.
+  1. Initialize |frame| as follows:
+    <dl class="switch">
+      <dt>If |session| has previously run an [=XR animation frame=], the user agent MAY:</dt>
```

Comment:
This MAY _could_ be a MUST, which would prevent content that stashes stuff on frames from behaving differently on different browsers (though content shouldn't really be relying on that anyway).

But we should definitely check with UAs before doing this